tests/Tests/500__jupyterhub/temp.robot

```diff
@@ -0,0 +1,42 @@
+*** Comments ***
```

Comment:
Could you rename temp.robot to a more meaningful filename?

Reply:
I have created this file temporarily to test the keyword and this file will be deleted once it gets the verified tag
I have written the same at line no.1 and 2